README.md

```diff
@@ -46,15 +47,15 @@ None
 
 | Variable | Default |
 |----------|---------|
-| `__java_packages` | `["java-1.8.0-openjdk"]` |
+| `__java_packages` | `["java-1.8.0-openjdk", "java-1.8.0-openjdk-devel"]` |
 
-## Ubuntu-14.04
+## Ubuntu-14
```

Comment:
reading your questions, i thought you did not understand my points because my explanation was not clear and i tried to explain them.
and i did not think i had to answer with "because".
here are my answers.
> why are you trying to change the meaning of the sections?

because i am not. i pointed out that "Ubuntu-14" can be understood as "Ubuntu-14.04 and Ubuntu-14.10" by users. i know you do not intend to send a wrong message, but it could be.


> why are you trying to be so specific while I am trying to write generic roles that could work for non-supported versions? what is the reason?

because i think the role should be generic but documents should not describe the untested versions.
the table above explains the detail.

i thought it is obvious and not necessary to make it a rule once you understand my points.